packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -413,24 +425,118 @@ contract Deploy is Deployer {
+
+        module_ = address(
+            new LivenessModule({
+                _safe: councilSafe,
+                _livenessGuard: LivenessGuard(guard),
+                _livenessInterval: cfg.livenessModuleInterval(),
+                _thresholdPercentage: cfg.livenessModuleThresholdPercentage(),
+                _minOwners: cfg.livenessModuleMinOwners(),
+                _fallbackOwner: fallbackOwner
+            })
+        );
+
+        save("LivenessModule", address(module_));
+        console.log("New LivenessModule deployed at %s", address(module_));
+    }
+
+    /// @notice Deploy a Security Council with LivenessModule and LivenessGuard.
+    function deploySecurityCouncilSafe() public broadcast returns (address addr_) {
+        console.log("Deploying Security Council Safe");
+        (SafeProxyFactory safeProxyFactory, Safe safeSingleton) = _getSafeFactory();
+
+        address[] memory initialSigners = new address[](1);
+        initialSigners[0] = msg.sender;
+
+        bytes memory initData = abi.encodeCall(
+            Safe.setup, (initialSigners, 1, address(0), hex"", address(0), address(0), 0, payable(address(0)))
+        );
+        Safe safe = Safe(
+            payable(
+                address(safeProxyFactory.createProxyWithNonce(address(safeSingleton), initData, block.timestamp + 1))
+            )
+        );
+
+        save("SecurityCouncilSafe", address(safe));
+        console.log("New SecurityCouncilSafe deployed at %s", address(safe));
+
+        address guard = deployLivenessGuard();
+        _callViaSafe({ _safe: safe, _target: address(safe), _data: abi.encodeCall(GuardManager.setGuard, (guard)) });
+        console.log("LivenessGuard setup on SecurityCouncilSafe");
+
+        address[] memory securityCouncilOwners = cfg.securityCouncilOwners();
+        for (uint256 i = 0; i < securityCouncilOwners.length; i++) {
+            _callViaSafe({
+                _safe: safe,
+                _target: address(safe),
+                _data: abi.encodeCall(OwnerManager.addOwnerWithThreshold, (securityCouncilOwners[i], 1))
+            });
+        }
+
+        // Now that the owners have been added, we can set the threshold to the desired value.
+        uint256 newThreshold = cfg.securityCouncilThreshold();
+        _callViaSafe({
+            _safe: safe,
+            _target: address(safe),
+            _data: abi.encodeCall(OwnerManager.changeThreshold, (newThreshold))
+        });
+
+        // Now that the owners have been added and the threshold increased we can deploy the liveness module (otherwise
+        // constructor checks will fail).
+        address module = deployLivenessModule();
+
+        // Unfortunately, a threshold of owners is required to actually enable the module, so we're unable to do that
+        // here, and will settle for logging a warning below.
+        addr_ = address(safe);
+        console.log("New SecurityCouncilSafe deployed at %s", address(safe));
```

Comment:
Any reason we change the threshold before we enable the module? Couldn't we enable the module then up the threshold? Or that doesn't work for some reason?

Reply:
The module's constructor checks this: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/Safe/LivenessModule.sol#L71

I'm not certain that check is strictly necessary, but I don't like the idea of removing it at this point.